pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -521,6 +522,11 @@ self: super: {
   # https://github.com/osa1/language-lua/issues/14
   language-lua = dontCheck super.language-lua;
 
+=======
+>>>>>>> c723352d7741f5b348be652b35e8c2f1a1908c22
```

Comment:
That diff doesn't look right. You seem to have leftovers from a previous merge in there.